packages/nextjs/test/config.test.ts

```diff
@@ -580,4 +590,28 @@ describe('Sentry webpack plugin config', () => {
       );
     });
   });
+
+  it.each([
+    /** `distDir` is not defined */
+    [getBuildContext('client', {}), '.next'],
```

Comment:
You’ll want to also include test name here so that test failures are easier to identify